modules/account-lib/src/coin/baseCoin/baseTransactionBuilderFactory.ts

```diff
@@ -19,4 +19,9 @@ export abstract class BaseTransactionBuilderFactory {
    * Returns an specific builder to create a wallet initialization transaction
    */
   public abstract getWalletInitializationBuilder();
+
+  /**
+   * Returns a specific builder to create a funds transfer transaction
+   */
+  public abstract getTransferBuilder();

```

Comment:
typically a factory takes like an enum and some params, I'll have to talk internally about this pattern, kinda want to re-factor this already but this looks good for now